webpack.config.js

```diff
@@ -171,6 +168,7 @@ if ( isProd ) {
 		},
 		mangle: {
 			screw_ie8: true,
+			except: [ 'translate' ],
```

Comment:
It would be better to keep all the "mangling" for `production` builds, but disable mangling `translate` it for the `i18n` task. Would that be too messy?